kube-client/src/config/incluster_config.rs

```diff
@@ -29,6 +37,43 @@ pub fn kube_dns() -> http::Uri {
     http::Uri::from_static("https://kubernetes.default.svc/")
 }
 
+pub fn try_kube_from_env() -> Result<http::Uri, Error> {
```

Comment:
I think this is much better than what we had. It should probably have at least a one-liner of docs if it is to be `pub` though.

Reply:
These aren't actually `pub`, since the module isn't exported. I'll narrow the visibility to `pub(super)`.